### DISCUSSION TITLE
建议增加手动导入直播源筛选功能

### DISCUSSION THREAD

**Discussion by piaoliuchong** [Guovin/iptv-api#755] [General]
1、目前直播源只能读取在线的，能不能手动建立直播源列表，直接从列表中筛选。
2、arm64环境docker安装lite版，服务器重启后，程序没自动启动，需要重新docker run启动。
3、怎样定时任务，怎样更新docker镜像，给个教程吧。

**Comment by piaoliuchong** [comment]
另外更新时间，能不能放到最后一行，放第一行，频道数字就变了，遥控器1按键对应不是cctv1了。对老人不友好。

**Reply by Guovin** [reply]
你可以修改配置移除显示更新时间

**Reply by piaoliuchong** [reply]
> 你可以修改配置移除显示更新时间

带着时间，可以看出来每天都在更新，给人一种稳定可靠直播源的感觉。

**Reply by Guovin** [reply]
v1.5.9 已支持调整更新时间位置

**Comment by Guovin** [comment]
1.后续会考虑增加订阅源的缓存使用；关于本地的直播源列表，可以将数据添加到config/demo.txt模板文件中，并开启使用历史结果，程序将会读取其中的频道数据进行处理。
2.对于服务器重启，容器需要你手动启动，这是程序无法控制的，一般服务器很少会出现重启的情况。
3.#440 目前定时任务是6点与18点执行，后续会增加修改配置；docker更新只要重新拉取镜像（docker pull）重新运行容器即可，对于配置更新，主机挂载的配置需要覆盖一次。

**Comment by dujq666** [comment]
使用该地址访问不到，浏览器也访问不了。



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2025年1月9日(周四) 上午10:04
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;
主题: Re: [Guovin/iptv-api] 建议增加手动导入直播源筛选功能 (Discussion #755)




 
v1.5.9 已支持调整更新时间位置

**Comment by Guovin** [comment]
已支持本地源功能，config/local.txt
